module/zfs/arc.c

```diff
@@ -1204,11 +1204,10 @@ hdr_full_dest(void *vbuf, void *unused)
 static void
 hdr_full_crypt_dest(void *vbuf, void *unused)
```

Comment:
This is not the same function as in #13194. That issue relates to v2.1.3, which is line 1217 on master.

Reply:
oh, that's fair enough actually